doc/source/ray-air/examples/torch_detection.ipynb

```diff
@@ -0,0 +1,1057 @@
4+AAx4z6rSxUiCEkjQlnHVdFwdhDEhBQ6b9iEUBwKaXfd/nWRaGodaKBWwwnAHvszAWTRvHcaulIwh5wDHRWkdpwiK2XC/G4zFBOMkzY20vxWApq6pqNBqlacYor8rmzumdiAfIIwJJnCTJaFT17cViLTSezG5XjS8rNRrPdrvqcP9kNj0kKJ6MDz3gzpO6kZjwXqrNruqsapTYlbU1Xva9d3BysE9C7p1Tbc8g3q4369Vqu9mkQTTO8s+98dnNzbrIUufcar3dlfVmtxaq/+DDjz5878OIB5dnF6oTQ3Le4yefFOORtuYHP/zjQUDgUfjwlZcRQnGU/tG3v7u3f1g2rfFACHV4eOytK9erl+7f++FPftIquz86WD2/xh5st1sYxNJ6L3WKCbaeQkQIqcsqDkOrNAUOAQugzopMWgcIL/LpbP9AYVAaEaaJdnZx8+Ll+3ew9UD7z3z6C5QnyvggTAHhtBg9rjdPbPNhv4G3JvggV16kbVtcrT+Hk/hyiUXXNaW3OuVhREMtzWg0QgQLrdq23Za1Mi6gHBiLlMXKFkkaBEEQBHXXrjbrs7MzCEAcx6vdptdSafns+dP1euM9oIQ56621fd8PmrIxznsohBpc3kEQKqOHeWmUxg74pmsHV0lVNWEYxnGolEKU3Ll/L80SY3UYhqPJRBi9rRsAQNs0xlnrHeUEEdiIvu67VnRhGHRN7ay5WS+vFter9bJpKgPswfGRFGJ/OvvKV77y+V/4/N37915//fWX798f5ZkbvDiUEEIYY3XbcM6GrSitNaG07To7zF0JGcakGGMPwTDVHPKd+74f9JyyrAa25ZAW/TMn5TBEHWSQYe12QB0MW7iDVDUcDIO2Y4yy2hCInFGDWOSBtd5IY//xP/9nFy8u81Fxdna2WKwo5bpTzx49/fiDj1mcn9y7/+Lmomy2LGEe41rZvjMx4sxBxghA/maxOrlzt7GKcI6cc0LkWVIUGQ6QD0Bl9bpv1nXlMTLe9VJQyoIglFJCD7zQjNDZ7dNsPp6M01EU5kFKEc+nezCMqo1QrS+K4vzyTNrOOqU7gT1Iw7BtGxiws+trHrIgCJq6gxBzzhEAUDnRiarrO2hREnZWWGC8t9gBTnhVGxJNZTBBe/eDW6/Rk1d0drSD8SfXVTI6GI0mEeWc0CBgla7CIsryWGnx9Pzi6YsX9155KcnSPM9Fq8ud6FrrtJtOJ8Lo69VSGx8GyXRy4CF8+Oqrr3/2jeO7t1CAcQi+98HNf/qf/3f/5kdPSHF6vdOfXKxaCSBmwJNWmhc3i0ZIFscOOgDAi7Nzp40FBNGAEDIbjzbLhZECUYKKLBtlOfCeR6EGruwaQLH1XkppoHfQcUJDxhGARunteh1FEaX0+Pjo6OiorVrRaylVXdfj8bhpGuecliaN0uH31tqsyJfbnYUAYRDnMQ34tiqd1fPppO17xthqtSrLUkqR5mkUhqLveRDEWYoJMcZQD7F2FOG+aZ21wPmhFwuiEELIGYMAAPvT1owx5iGwEPAgcM4RRpfrlZByb3/eNM3QmEgpsyxjmADtbac5plpILdWnXnutqirOw1vHtwLG27ZfLJdxmuztHxyenG6qajbfh4hiGk73j777vR94Bx3wPOQnJ8fz+QElvO80hDiIwqppTk5OoihK05SHofHOaKeVublexmGCEHn+/HwynjV1d3bxoqqqsiwno/HZ2VmUxLP53ma3lVo9PXs+2Zvno7GU8uDgoBiPpnszbV0cxx9++OF7771XjEcXly/+4A/+4L333ltvN6vN+vL66hd/8Re/873vllVDKd1tq/c/+PDp06cnR8fvvP3jqm0+evS4rXpkMfVwEmdGOwAxxaSvGgBAHIc8CJIshwbItgtD3ve1kA0AZm8+BQAwhEMWzg73bz98eHLn7nQ2m+5PpRbXi6vFYrHebpIkS6OUUpoU+a5veD7i8/2Pu/rMyUfVdTxNZnH01sFR++77ny2mIyHHhBDjkUfAQWfM9fW1tRp5F8exhzBMkk25IwgHhKqmc1K3Vb3d7Zz3g9tXCEEw7rru/0f0hnBYL/qZTDFo64P0PFiSB+UaIRTH8c8ALJTSAQpICCKMbtbl/v7eyy+/vNlsBurvkMRU1W2QREmWxkmitaaYWOAb0cdp0kvhve+6hhDigNVaW2Awhh999FEQRFEUtW2z3W6+853vKGUQQl3fDvV3PB4nWToMXYe43WHsOXzRhiZdCNE03aCqD+4dhNAgtQ+OdSnlsJsahsHAsBwmCoMWX9f10LMPkKyhjkMI0zQFAEwmk8Em9DPNffBWIoTqunTOCCEg8sP15eBw/4MPPnj69Olms/UOFFluta2rLkuy2XS/Kpu2G04Ruz8/GBShvdm+UZoi7Ixdr9dBEKwXS044gmB4xIgHCPoooHkGlQGUsygItZQYY0ypAx5jOp8fdF1XFMXx8fF2uw2icJivxDx02tV9d+/hSwd7B86Y+Xye5HEch1qJgPG9yXSwzRhr5wf7690WQqiM6YQyxjDCKKYU4yhOKQ+F0gZAHicwiBqL+ORAkPSi0nhy9LwSgmd0clRDFk33ksmsFvLTn/s5iEEcBXmW3nvwYDydLG8WRZyeHh4o2S93m72To9F4mo/GSqm3//iHUsq6bjDEWZZ7760D2+3WGW+tPdg/irP89v2HJE41AT3Gv/v1H/+//s4/OC81yPd2Bl6uqt4AC9H04EAIsdvtgiDgnMdhslltKeXKGM756enpbDYjhCCAfN+31IJmW1dtBygr21JriTGiAbEe1r1Yr7dd0xqrbt0+eeONNyIehox578uywpjEYeIt4JwuVzeccwyxFEZKFQRcKUkY3rY1SwJh9fX1tdZaaGWBdxicL6+s90KpNI4ZgZDTsm+jKMIA7nY7i2FpRG1kgDl2qNMynY2RB5wxqRSipG4aKaXTJuFxwKObxSpMk5vNylrrIJDIV0Z2UozHY4xxtSul6CjDhFLjHXDeaUO1u71/5KQZF
```

Comment:
does this take 25 min to finish 🤨. how many cpus did you have in your cluster?

is `read_images` faster? if so, we should maybe use the same implementation as that (`ImageDataSource._read_file`)

Reply:
It's 25 seconds.